discovery-provider/alembic/trigger_sql/handle_repost.sql

```diff
@@ -105,6 +105,55 @@ begin
       on conflict do nothing;
     end if;
 
+	-- notify followees of the reposter who have reposted the same content
+	-- within the last month
+	if new.is_delete is false
+	and new.is_repost_repost is true then
+	with
+	    followee_repost_repost_ids as (
+	        select user_id
```

Comment:
did you find out the bug in the demo? maybe use distinct here in case

Reply:
yea i didnt filter out deleted and non current reposts but i added that below